.changeset/fair-falcons-search.md

```
@@ -0,0 +1,5 @@
+---
+'@siemens/ix': patch
+---
+
+Improved **ix-menu** accessibility by implementing the w3c menubar pattern, adding better keyboard navigation and screenreader suppport.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)